portfolio/src/main/webapp/script.js

```diff
@@ -43,10 +42,32 @@ function deleteBuffer() {
       loginLink.href = responseJson.url;
       alertCard.appendChild(loginLink);
     }
-    alertCard.style.display = 'block';
+    showAlertMode(alertCard);
   })
 }
 
+/**
+ * Helper to show alert panel and blacked out background
+ */
+function showAlertMode(alertCard) {
+  alertCard.classList.add('active');
+  const backdrop = document.getElementById('backdrop');
+  backdrop.classList.add('active');
+  backdrop.addEventListener('click',() => {
+    hideAlert();
+  })
```

Comment:
I think this can be `addEventListener('click', hideAlert);`

Reply:
Oh that works! I tried ('click', hideAlert()) before, but that just executes hideAlert immediately. I didn't know the () made a differencde.